stock_request_mrp/tests/test_stock_request_mrp.py

```diff
@@ -1,31 +1,52 @@
-# Copyright 2016-20 ForgeFlow S.L. (https://www.forgeflow.com)
-# Copyright 2022 Tecnativa - Pedro M. Baeza
-# Copyright 2022 Tecnativa - Víctor Martínez
+# Copyright 2016-24 ForgeFlow S.L. (https://www.forgeflow.com)
+# Copyright 2022-24 Tecnativa - Pedro M. Baeza
+# Copyright 2022-24 Tecnativa - Víctor Martínez
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
 
 from odoo import fields
-from odoo.tests import Form
+from odoo.tests import Form, common, new_test_user
 
-from odoo.addons.stock_request.tests.test_stock_request import TestStockRequest
 
-
-class TestStockRequestMrp(TestStockRequest):
```

Comment:
Any reason why you no longer can reuse TestStockRequest?

Reply:
Initially, I found an issue with the attributes defined in TestStockRequest not being initialized correctly, which led to an error when accessing them in the TestStockRequestMrp class, so I replicated the necessary setup code directly in TestStockRequestMrp as i saw in this pr https://github.com/OCA/stock-logistics-request/pull/24.
However, you are right, the proper approach would be to reuse the TestStockRequest, i have now fixed it to follow this approach.